modules/apps/transfer/ibc_module.go

```diff
@@ -96,15 +96,14 @@ func (im IBCModule) OnChanOpenTry(
 	channelID string,
 	chanCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
-	version,
 	counterpartyVersion string,
-) error {
-	if err := ValidateTransferChannelParams(ctx, im.keeper, order, portID, channelID, version); err != nil {
-		return err
+) (string, error) {
+	if err := ValidateTransferChannelParams(ctx, im.keeper, order, portID, channelID); err != nil {
```

Comment:
You still need to ensure that counterparty version is correct

Reply:
the line after this checks the counterparty version

Reply:
🤦 my bad!